lib/nixos.nix

```diff
@@ -0,0 +1,60 @@
+# SPDX-FileCopyrightText: 2020 Serokell <https://serokell.io/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+###
+# Utilities for configuring and building NixOS
+###
+
+{ pkgs }:
+
+rec {
+  # Evaluate a NixOS configuration.
+  #
+  # config: NixOS configuration.
+  # modules: extra modules to add
+  buildSystem = { config, modules ? [ defaults ] }:
+    let
+      c = import (pkgs.path + "/nixos/lib/eval-config.nix") {
+        modules = modules ++ [ config ];
+      };
+    in c.config.system.build.toplevel;
+
+  # A module with reasonable NixOS configuration defaults.
+  # Note that you don’t need to pass any arguments to it.
+  defaults = { pkgs, lib, ... }: {
+    nixpkgs.localSystem.system = "x86_64-linux";
+
+    documentation.nixos.enable = false;
+
+    nix = {
+      # https://github.com/NixOS/nix/issues/1964
```

Comment:
This issue is resolved

Reply:
Well, I just copy-pasted it 🤷. Is it resolved in all pins that we use?

Reply:
I don't really know, I guess it doesn't hurt to have the parameter